### ISSUE TITLE
Create command-line executable

### ISSUE THREAD

**Issue by kmfisch** [ucsd-ccbb/VAPr#16]
Create the option to run from the command-line in addition to through the jupyter notebook

**Comment by AmandaBirmingham** [comment]
I do not immediately see how to do this in a useful way due to the large number of parameters that need to be provided.  To run annotate, we need to capture from the user:
* input_dir
* output_dir
* mongo_db_name
* mongo_collection_name
* annovar_install_path
* an optional design_file
* an optional genomic build version
* optional information on whether the vcfs provided are bgzipped or not
* an optional number of processes to run
* an optional chunk size
* an optional verbosity level
* an optional flag to allow adding data to an existing dataset

That is a heck of a lot of command line parameters; although we could theoretically leave all the optional ones at their defaults, that would make the command-line option pretty un-useful.  We could theoretically store the parameters in a config file that would be read when the annotation process was run from the command line, but that is a non-trivial refactor. 

The bottom line is that I don't understand how we envision VAPr being used from the command line. @kmfisch Please provide either a description of the use-case(s) in which you would use this functionality (so I can intuit a sensible interface) or specify the command line interface you want.
